Samples/HttpTrigger2/main.py

```diff
@@ -0,0 +1,5 @@
+import azurefuncs
+
+
+def main(req: azurefuncs.HttpRequest):
```

Comment:
Is the type hint here required? Wondering if we can decipher it using the `type` property in `function.json`.